dawn/src/dawn/CodeGen/CXXNaive-ico/CXXNaiveCodeGen.cpp

```diff
@@ -261,14 +263,17 @@ void CXXNaiveIcoCodeGen::generateStencilWrapperCtr(
           auto hdims = sir::dimension_cast<sir::UnstructuredFieldDimension const&>(
               field.getFieldDimensions().getHorizontalFieldDimension());
 
-          auto getNumElCall = [](const sir::UnstructuredFieldDimension& hdims) -> std::string {
+          auto getNumElCall = [&](const sir::UnstructuredFieldDimension& hdims) -> std::string {
```

Comment:
The value is correct. However since it's no longer representing the number of elements, but rather the size of the horizontal dense dimension the name `getNumElCall` is a bit misleading.